kqueen/engines/eks.py

```diff
@@ -70,18 +70,18 @@ class EksEngine(BaseEngine):
             }
         },
         'cluster': {
-            'node_count': {
-            # TODO unsupported, need to define template k8s input
-                'type': 'integer',
-                'label': 'Node Count',
-                'default': 3,
-                'order': 3,
-                'validators': {
-                    'required': True,
-                    'min': 1,
-                    'number': True
-                }
-            },
+#            'node_count': {
```

Comment:
I also suggest to here add link  to the issue, that we are waiting to be resolved

Reply:
seems there are no link, we have not got api support of cluster resizing